pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManagerConfig.java

```diff
@@ -50,6 +50,8 @@
   public static final String INSTANCE_SEGMENT_TAR_DIR = "segmentTarDir";
   // Key of segment directory
   public static final String INSTANCE_BOOTSTRAP_SEGMENT_DIR = "bootstrap.segment.dir";
+  // Key of segment store uri
+  public static final String SEGMENT_STORE_URI = "segment.store.uri";
```

Comment:
+1

Reply:
Since import HelixInstanceDataManagerConfig's "segment.store.uri" would cause circular dependency between pinot-server and pinot-core, I added this config key to CommonConstants.